.github/workflows/yocto-build-deploy.yml

```diff
@@ -814,9 +818,21 @@ jobs:
           fi
           #[ "${VERBOSE}" = "verbose" ] && _debug="--debug"
 
+          # create docker-compose.yml with OS release metadata for the hostapp
+          cat > "${WORKSPACE}/docker-compose.yml" <<EOF
+version: '2.4'
+services:
+  hostapp:
+    image: ${_local_image}
+    labels:
+      io.balena.image.store: 'root'
+      io.balena.image.class: 'hostapp'
+      io.balena.update.requires-reboot: '1'
+      io.balena.private.hostapp.board-rev: '${DEVICE_REPO_REV}'
+EOF
```

Comment:
This indentation needs to be aligned with the level of the other commands above like `cat > ...` in order to be parsed as valid yaml. When it is executed it will still behave as expected and not have any leading whitespace in the compose file.